src/main/java/com/minecolonies/coremod/colony/ColonyView.java

```diff
@@ -39,79 +39,71 @@
 public final class ColonyView implements IColony
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') class "ColonyView" has 60 methods, which is greater than the 35 authorized. Split it into smaller classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1448)